docs/content/tutorial/step_09.ngdoc

```diff
@@ -38,16 +38,16 @@ Since we are using [Bower][bower] to install client-side dependencies, this step
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "angular": "1.5.x",
-    "angular-mocks": "1.5.x",
-    "angular-route": "1.5.x",
+    "angular": "1.6.x",
+    "angular-mocks": "1.6.x",
+    "angular-route": "1.6.x",
```

Comment:
I don't think these changes (here and elsewhere) are correct. The tutorial still uses 1.5.x iirc.